extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/ChangeRecord.java

```diff
@@ -154,6 +157,48 @@ public interface ChangeRecord {
     @Nonnull
     RecordPart value();
 
+    /**
+     * Returns the new value of the record. For <em>sync</em>, <em>insert</em> and <em>update</em> operations the value
+     * describes the database record as it looks AFTER the event, so the latest image, for <em>delete</em> it returns null.
+     */
+    @Nullable
+    default RecordPart newValue() {
+        try {
+            switch (operation()) {
+                case SYNC:
+                case INSERT:
+                case UPDATE: return value();
+                default: return null;
+            }
+        } catch (ParsingException e) {
+            throw rethrow(e);
+        }
+    }
+
+    /**
+     * Returns the new value of the record. For <em>update</em> and <em>delete</em> operations the value
+     * describes the database record as it looks <strong>before</strong> the event,
+     * for <em>sync</em> and <em>insert</em> it returns null.
+     */
+    @Nullable
+    default RecordPart oldValue() {
+        try {
+            return operation() == Operation.DELETE ? value() : null;
+        } catch (ParsingException e) {
+            throw rethrow(e);
+        }
+    }
+
+    /**
+     * Returns {@link #newValue()} as JSON.
+     */
+    String getNewValueJson();
+
+    /**
+     * Returns {@link #oldValue()} as JSON.
+     */
+    String getOldValueJson();
```

Comment:
Do we need these? One can call `newValue().toJson()`, this doesn't seem to bring much benefit.

Reply:
I agree, just wanted the API to be consistent with previous. I'm ok to remove it.